lib/src/utils/ui_constants.dart

```diff
@@ -45,4 +45,12 @@ abstract class UiConstants {
   static const double movieTitleFontSize = 30.0;
   static const double releaseDateFontSize = 20.0;
   static const double voteAverageFontSize = 36.0;
+
+  // back_button
+  static const double paddingBackButtonDetails = 10.0;
+
```

Comment:
this empty line could be removed to keep consistency

Reply:
Done